src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +15,63 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't insert Content entity", e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Couldn't get comment", e);
+        }
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        try (Session session = factory.openSession()) {
+            Query<Comment> getAllCommentsQuery = session.createQuery("from Comment", Comment.class);
+            return getAllCommentsQuery.getResultList();
+        } catch (Exception e) {
+            throw new RuntimeException("Couldn't get all comments", e);
+        }
     }
 
     @Override
     public void remove(Comment entity) {
-
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.remove(entity);
+            transaction.commit();
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Couldn't insert entity", e);
```

Comment:
```suggestion
            throw new RuntimeException("Couldn't remove entity", e);
```
same for the other remove() methods